src/modules/message/store/action.js

```diff
@@ -46,6 +46,28 @@ export const signMessage =
 export const signClaimMessage =
   ({ nextStep, portalMessage, privateKey, currentAccount }) =>
   async () => {
+    if (currentAccount?.hw) {
+      const [error, signature] = await to(
+        signClaimMessageUsingHW({
+          account: currentAccount,
+          message: getUnsignedNonProtocolMessage(portalMessage),
```

Comment:
For the claim message, it shouldn't add any tag

Reply:
Realized yesterday and fixed. I'll test and push.